_posts/build_fdn/2020-06-09-Setup-Commandline-for-Realm-Import.md

```diff
@@ -9,7 +9,7 @@ site: build
 The commandline interface to import the Realm app code is called "realm-cli". We will set this up in this step, then import all the Realm code in the next step.
 
 1. realm-cli setup
-   1. Install realm-cli - follow the instructions for manual install or `npm` install at <A HREF="https://docs.mongodb.com/realm/deploy/realm-cli-reference/" target="_blank">https://docs.mongodb.com/realm/deploy/realm-cli-reference</A><BR><BR>
+   1. Install realm-cli - follow the instructions for manual install or `npm` install at <A alt="cli install link" HREF="https://docs.mongodb.com/realm/deploy/realm-cli-reference/" target="_blank">https://docs.mongodb.com/realm/deploy/realm-cli-reference</A><BR><BR>
```

Comment:
Hi! regular links - <A> tags - do not need alt tags. Only images do, thanks! Please remove this change. Everything else looks good, will merge when the alt tag on this link is removed.

Reply:
@Sheeri thanks!, alt tag has been removed, should be good to go. Thanks!